shepherd/server.py

```diff
@@ -47,136 +43,38 @@ def connect():
 @socketio.on('join')
 def handle_join(client_name):
     print(f'confirmed join: {client_name}')
+    join_room(client_name)
```

Comment:
add a comment saying what the two-room options are and how they need to be passed in

Reply:
also it appears this hasn't been done in scoreboard

Reply:
removed this logic, so now `handle_join` just prints as usual